database/entity/UserContact.ts

```diff
@@ -1 +1 @@
-export { UserContact } from './0049-add_user_contacts_table/UserContact'
+export { UserContact } from './0053-change_password_encryption/UserContact'
```

Comment:
Why?

Reply:
path traversal references along code, User needs to go with the hand with UserContact